ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -182,6 +185,16 @@ router bgp {{ router_bgp.as }}
       route-target export {{ route_target }}
 {%             endfor %}
 {%         endif %}
+{%         if router_bgp.vlans[vlan].route_targets.import_evpn_domains is arista.avd.defined %}
+{%             for route_target in router_bgp.vlans[vlan].route_targets.import_evpn_domains %}
+      route-target import evpn domain {{ route_target.domain }} {{ route_target.route_target }}
+{%             endfor %}
+{%         endif %}
+{%         if router_bgp.vlans[vlan].route_targets.export_evpn_domain is arista.avd.defined %}
+{%             for route_target in router_bgp.vlans[vlan].route_targets.export_evpn_domain %}
+      route-target export evpn domain {{ route_target.domain }} {{ route_target.route_target }}
```

Comment:
Can we do something like:

  route-target import export evpn domain remote BLA:BLA

This is a common configuration for gateways instead of having multiple lines per MAC-VRF.

Reply:
If it is available in CLI we could. Any other abstraction should be done in eos_designs only.

Reply:
This cli doesn't exist on my box under MAC VRF. It's either route-target both BLA:BLA (no domain can be specified) or route-target import evpn domain remote BLA:BLA + route-target export evpn domain remote BLA:BLA.

Reply:
I agree with the automation getting it out of the way, but it's very useful if you have many MAC-VRFs as it reduces the config size. I have tested that in the lab, unicast me to check the details please.

Reply:
I stand corrected, the box that i tried this on before must have had a bad day. Added this feature.

Reply:
default route injections on ip vrf should be in a different PR imo.